readme.md

```diff
@@ -1,4 +1,13 @@
 # Marvin
+
 The new Wikipedia mobile website. Read more on our [wiki]!
 
-[wiki]: https://www.mediawiki.org/wiki/Reading/Web/Projects/NewMobileWebsite
\ No newline at end of file
+## Development
```

Comment:
Added link to the changelog in f8a8da4, lets do license when we include the license file in the repo.

I've created https://phabricator.wikimedia.org/T172682 to not forget about the license.